generators/typescript/sdk/client-class-generator/src/endpoints/utils/generateHeaders.ts

```diff
@@ -86,7 +89,7 @@ export function generateHeaders({
         );
     }
 
-    context.importsManager.addImportFromRoot("src/core/headers.js", {
+    context.importsManager.addImportFromRoot(`${exportsManager.packagePath}/core/headers.js`, {
```

Comment:
This looks weird to me. I'd expect the imports manager to receive the package path and already be aware of it, so that the existing calls to importsManager don't require updating.